src/Layers/xrRender_R2/r2_R_lights.cpp

```diff
@@ -129,14 +129,14 @@ void CRender::render_lights(light_Package& LP)
         }
     };
 
-    const auto& flush_lights = [&]()
+    const auto& flush_lights = [] (CRender& render, xr_vector<task_data_t>& lights_queue)
```

Comment:
I guess it's doubtful, but I suppose a lambda without capture is more likely to be inlined.

Reply:
CRender is being instantiated only once as a global variable called RImplementation.
Instead of passing it as CRender& render variable, it's better to just capture this.

Reply:
> I guess it's doubtful, but I suppose a lambda without capture is more likely to be inlined.

Ah, I see. But what if we capture only `this`?
And also I'm not sure if inlining helps here.